components/brave_rewards/resources/ui/reducers/grant_reducer.ts

```diff
@@ -71,7 +71,7 @@ const grantReducer: Reducer<Rewards.State | undefined> = (state: Rewards.State,
       }
 
       state.currentGrant = currentGrant
-      chrome.send('brave_rewards.getGrantCaptcha', [])
+      chrome.send('brave_rewards.getGrantCaptcha', [currentGrant.promotionId, currentGrant.type])
```

Comment:
in extension you used checks for `promotionId` and `type`. Should we use it here as well?

Reply:
I can add that. It was added in extension because compiler threw error there. Doesn't seem to mind here as they are passed in as elements.

Reply:
Done.